src/Sarif.Viewer.VisualStudio.UnitTests/Converters/CallTreeNodeToTextConverterTests.cs

```diff
@@ -106,39 +90,19 @@ public void CallTreeNodeToTextConverter_HandlesNonCallOrReturnNodes()
                 }
             };
 
-            VerifyConversion(callTreeNode, "Continuation");
+            VerifyConversion(callTreeNode, Microsoft.Sarif.Viewer.Resources.UnknownCalleeMessage);
```

Comment:
Why "unknown callee"? That assumes that the code flow step was a function call. Why not just "Continuation" or "Continuing..."?

Reply:
Changed